packages/peregrine/lib/talons/MiniCart/useKebab.js

```diff
@@ -0,0 +1,27 @@
+import { useCallback, useRef, useState } from 'react';
+import { useEventListener } from '@magento/peregrine';
+
+export const useKebab = () => {
+    const kebabRef = useRef(null);
```

Comment:
This is great. Coincidentally, it's also literally a full reimplementation of `useDropdown`, in which the only differences are the inclusion of a callback to expand (`handleKebabClick`) and attachment of an extra listener (`touchend`).

As I've mentioned before, creating talons is going to reveal real opportunities for code reuse. 😄 

We can probably keep `useKebab` for now, but I would add a to-do comment.

Reply:
Added a comment.